### PR TITLE
Pre-release Themis 0.15.0

### PR DESCRIPTION
**Pull request by G1gg1L3s**
Currently the tag is on the pre-release branch, so we need to remove it and put again. But for now, let's just run the tests again.

## Checklist

- [x] Change is covered by automated tests

### CODE REVIEW

**Comment by G1gg1L3s** [comment]
Seems like we need to also modify hashes here:

https://github.com/cossacklabs/themis/blob/bc94b63eb6f46f46d82b164264f3497f3a98bf9d/PKGBUILD.MSYS2#L20-L22

**Comment by G1gg1L3s** [comment]
I added the `gothemis/v0.15.0` tag to run the checks on buildbot. We will need to move it later, so it's a reminder.

**Review by Lagovas — APPROVED** [review]

**Comment by G1gg1L3s** [comment]
I've also moved the `0.15.0` to the latest commit, so these hashes are not correct again and therefore the MSYS2 tests will fail. We will adjust them after the merge (and then probably again after the release).

https://github.com/cossacklabs/themis/blob/7f4f406e2a94db7af138d16ae197c8c20750c7aa/PKGBUILD.MSYS2#L20-L22

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,13 +4,31 @@
...
 
+## [0.15.0](https://github.com/cossacklabs/themis/releases/tag/0.15.0), June 21st 2023
+
+**TL;DR:**
+
+- Uncompressed EC public keys are now supported.
+- Increased PBKDF2 iteration count from 200000 to 314110 for Secure Cell passphrase mode.
+- OpenSSL 3.0 is now supported.
+- Pythemis now uses `pyproject.toml`.
+- And as usual: enhanced security measures and fixed bugs.
```
Comment (vixentael) [inline CHANGELOG.md:15]: nice :)

**Review by vixentael — APPROVED** [review]

**Comment by vixentael** [comment]
Bless you!
giphy


However `Themis Core / MSYS2 environment` tests are Failing after 2m

**Pull request merged**